sound/soc/intel/boards/sof_sdw.c

```diff
@@ -351,6 +351,14 @@ static struct sof_sdw_codec_info codec_info_list[] = {
 		.init = sof_sdw_rt711_init,
 		.exit = sof_sdw_rt711_exit,
 	},
+	{
+		.part_id = 0x711,
+		.version_id = 2,
+		.direction = {false, true},
+		.dai_name = "rt711-aif2",
+		.init = sof_sdw_rt711_init,
+		.exit = sof_sdw_rt711_exit,
```

Comment:
that is more surprising, we should have an init function for the headset and one for the mic. Having the same init function for both seems weird.

We would also need a be_id or something that is unique so that we match with the topology